tests/integ/test_neo_mxnet.py

```diff
@@ -165,7 +159,7 @@ def test_inferentia_deploy_model(
 
         model.compile(
             target_instance_family=inf_instance_family,
-            input_shape={"data": [1, 1, 28, 28]},
+            input_shape={"data": [1, 1, 28, 28], "softmax_label": [1]},
```

Comment:
This isnt needed for Inferentia compilation flow. But if it works thats ok, it doesnt hurt anything.